core/src/styles/core/_dialog.scss

```diff
@@ -54,6 +54,11 @@ $-dialog-padding: 1.5rem;
   }
 }
 
+.cat-dialog-pull-out {
```

Comment:
No, I'd use the same ones and just make sure that they apply correctly. So If cat-dialog-pull is used within header it would set paddings to `-dialog-padding` and if it is used in the content it basically applies `0 -dialog-padding`.